blenderproc/python/loader/URDFLoader.py

```diff
@@ -0,0 +1,270 @@
+from typing import List, Union
+import os
+from mathutils import Matrix, Vector
+
+import bpy
+
+from blenderproc.python.utility.SetupUtility import SetupUtility
+from blenderproc.python.utility.BlenderUtility import get_all_materials
+from blenderproc.python.utility.Utility import Utility
+from blenderproc.python.loader.ObjectLoader import load_obj
+from blenderproc.python.material import MaterialLoaderUtility
+from blenderproc.python.types.MaterialUtility import Material
+from blenderproc.python.types.MeshObjectUtility import MeshObject, create_primitive
+from blenderproc.python.types.URDFUtility import URDFObject, Link, Inertial
+
+
+def load_urdf(urdf_file: str) -> URDFObject:
+    """ Loads a urdf object from an URDF file.
+
+    :param urdf_file: Path to the URDF file.
+    :return: URDF object instance.
+    """
+    # install urdfpy
+    SetupUtility.setup_pip(user_required_packages=["urdfpy==0.0.22", "networkx==2.5.1"])
+    from urdfpy import URDF
+
+    # load urdf tree representation
+    urdf_tree = URDF.load(urdf_file)
+
+    # create links
+    links = load_links(urdf_tree.links)
+
+    # recursively assign transformations depending on the local joint poses
+    for i, joint_tree in enumerate(urdf_tree.joints):
+        child = links[i + 1]
+        parent = links[i]
+
+        # convenient insanity check
+        assert child.get_name() == joint_tree.child
+        assert parent.get_name() == joint_tree.parent
+
+        # we also add some information to the link
+        child.set_joint_type(joint_type=joint_tree.joint_type)
+
+        # traverse local poses
+        child.blender_obj.matrix_world = parent.blender_obj.matrix_world @ Matrix(joint_tree.origin)
```

Comment:
This should be written with our api methods
```suggestion
        child.set_local2world_mat(parent.get_local2world_mat() @ joint_tree.origin)
```